desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -410,12 +404,31 @@ def test_handle_raz_ha(self):
           verify=False
         )
         assert_equal(raz_response.status_code, 200)
+        assert_equal(requests_post.call_count, 1)
+        requests_post.reset_mock()
 
-        # When no RAZ URL is healthy
+        # HA mode - When RAZ instance1 is unhealthy and RAZ instance2 is healthy
+        requests_post.side_effect = [Mock(status_code=404), Mock(status_code=200)]
+        client = RazClient(self.raz_urls_ha, 'kerberos', username=self.username, service="s3", service_name="cm_s3", cluster_name="cl1")
```

Comment:
is it not possible to use one RAZclient or call a method instead of writing it again and again?

Reply:
Done